Bottom: 48ee9ea23657f104e37bb65cdff1f031f5a7f94e
Top:    18117218216d3c6f236a7ef9bd66e9622fc9acae
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:41:46 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index d188f21..1c85c26 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1510,12 +1510,12 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
 
   /* DEBUG */
 //  poly_M_Copy0 (&PCB->Data->outline, info.poly);
 //  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  gui->invalidate_all ();
 
 //  poly_Free (&info.poly);
